### ISSUE TITLE
Support AWS SecretsManager versioning

### ISSUE THREAD

**Issue by kevbook** [tellerops/teller#117]
## Feature Request

**Describe the solution you'd like**
In AWS SecretsManager, there is a concept of versioning. It's supported by Go SDK as well.

AWS container orchestration with ECS fargate, we are using versioning, so Teller will be in-sync.  https://docs.aws.amazon.com/AmazonECS/latest/userguide/specifying-sensitive-data-secrets.html

```bash
aws get-secret-value --secret-id=abc --version-id=90D71E3C-ABCD-ABCD-ABCD-565F92429B23
```

Example:

```yaml
# Suggested Syntax
aws_secretsmanager:
    env_sync:
        path: vault-name
        path_version:
    env:
      VAR1:
        path: vault-name
        path_version:
```

**Comment by kaplanelad** [comment]
Hey @kevbook, sound a good feature.
Added this support in open PR: #121

**Comment by kaplanelad** [comment]
@kevbook  the fix just released in version v1.5.3
To fetch a spisific version your need to add a comma separating in path value (`path: prod/foo/bar,<VERSION>`).
